src/Airmap/AirMapAdvisoryManager.cc

```diff
@@ -74,7 +74,6 @@ AirMapAdvisoryManager::_requestAdvisories()
     _advisories.clearAndDeleteContents();
 
     Advisory::Search::Parameters params;
-    params.authorization = _shared.loginToken().toStdString();
```

Comment:
this is a bit of a mystery for me; advisory.h [never had](https://github.com/airmap/platform-sdk/commits/master/include/airmap/advisory.h) an `authorization` member in its `Advisory::Search::Parameters`